### ISSUE TITLE
Request: Can we get a 'continue generation' button?

### ISSUE THREAD

**Issue by bbecausereasonss** [brianpetro/obsidian-smart-connections#211]
I end up having to type this manually over and over, with the relatively short context size of the chat. Can we please have a continue generation button?

**Comment by fabioscarsi** [comment]
Absolutely, this is one of the most boring things about using Smart Connection's chat. A setting option that makes responses complete instead of fragmented would be a giant step forward.

**Comment by bbecausereasonss** [comment]
Bump @brianpetro :)

**Comment by brianpetro** [comment]
@bbecausereasonss changing to the 16K model (available in the settings) will help this.

While there are some strategies to implement a "continue" feature, it's probably better to wait for further context size increases. Since this was first posted(a month ago) the length of responses has already grown 4X.

Besides the continue solution, there is also enabling the chat to recursively call itself, similar to the ChatGPT Plugins/Functions. For example, if you need such a long response, asking GPT to plan the output into manageable steps and then executing one at a time. This type of feature is more the direction I'm working towards, and it would inherently solve the "continue problem."

I appreciate the feedback as it helps me consider all the angles in making Smart Connections the smartest it can be.

Thanks for your help
Brian 🌴

**Comment by brianpetro** [comment]
@bbecausereasonss is this still an issue or can we close this?

**Comment by brianpetro** [comment]
recorded response https://youtu.be/ORbmZgmfjKw
